corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -419,6 +424,21 @@ var AdvancedCase = (function () {
         };
     };
 
+    var ParentIndex = function () {
+        var self = this;
+
+        self.tag = ko.observable();
+        self.reference_id = ko.observable('parent');
+        self.relationship = ko.observable('child');
+        self.relationship.subscribe(function (rel) {
+            if (rel == 'extension' && self.reference_id() == 'parent') {
```

Comment:
Should use `===`. Also no real need for the second check, you could always set the reference_id even if its already set to the correct value.